app/backend/src/couchers/errors.py

```diff
@@ -44,6 +44,7 @@
 INVALID_MESSAGE = "Invalid message."
 INVALID_NAME = "Name not supported."
 INVALID_RECIPIENTS = "Invalid recipients list."
+INVALID_REGION = "Invalid region code."
```

Comment:
Change to "Invalid region.", the "code" will confuse users if they ever see this